code/api/src/migrations/5-updateUsersTable.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return Promise.all([
+      queryInterface.addColumn(
```

Comment:
I see that here instead of adding an entire table, you're just adding columns to the table and below is the inverted process.